tests/e2e/plugins/sitekit-mock.php

```diff
@@ -0,0 +1,120 @@
+<?php
```

Comment:
Let's add `<exclude-pattern>tests/e2e/plugins/*</exclude-pattern>` to the `phpcs.xml.dist`